frontend.go

```diff
@@ -15,18 +15,40 @@
 package casbin
 
 import (
+	"bytes"
 	"encoding/json"
+	"fmt"
+	"strings"
 )
 
 func CasbinJsGetPermissionForUser(e IEnforcer, user string) ([]byte, error) {
-	policy, err := e.GetImplicitPermissionsForUser(user)
-	if err != nil {
-		return nil, err
+	model := e.GetModel()
+	s := strings.Builder{}
```

Comment:
@closetool plz define `Model.ToText()` method in the model class to get the CONF content from a model struct, return value is string.

Then use it here.